client-java-android/README.adoc

```diff
@@ -1,22 +1,80 @@
-= FeatureHub SDK for Android/Polling
+= FeatureHub SDK for REST
 
 == Overview
-This SDK is intended for client libraries, particularly for Android as keeping the radio on would drain the battery
-quickly. It does this by making GET requests for the data  but sets a period for which it will hold off making new requests (a timeout, which you can set to 0 if you have your own timer). This will allow
-you to keep requesting updates but they will not actually issue calls unless the context changes
-or the timeout has occurred. 
+This SDK is intended for client libraries, e.g.:
 
+- Android as keeping the radio on would drain the battery quickly
+- lambdas or cloud functions where control over the HTTP request object is desired and you only need to get the state once during the lifetime.
+- other situations where updating the state of the internal repository is intermittent or desired to be consistent 
+
+The REST SDK *does not poll*. It allows the user of the SDK to create a polling mechanism which suites their application. 
+
+It fits into the FeatureHub Java SDK by providing a standard interface to make HTTP GET requests for the data  but sets a period for which it will hold off making new requests (a timeout, which you can set to 0 if you have your own timer), i.e. you can request data but it will ignore your request if it is within the timeout period. 
+
+This will allow you to keep requesting updates but they will not actually issue calls unless the Client Context changes or the timeout has occurred.
```

Comment:
```suggestion
This will allow you to keep requesting updates but they will not actually issue REST calls to the FeatureHub Edge server unless the Client Context changes or the timeout has occurred.
```